doc/build-unix.md

```diff
@@ -69,7 +69,7 @@ Build requirements:
 
 Options when installing required Boost library files:
 
-1. On at least Ubuntu 14.04+ and Debian 7+ there are generic names for the
+1. On at least Ubuntu 14.04+ ( > kernel 3.13 )  and Debian 7+( > kernel v3.2.41)  there are generic names for the
```

Comment:
I understand the idea is to index, to make it easier but it seems to other will become more confusing.

so I added this information, because in my opinion what makes Ubuntu 04.14 to be himself is your kernel, the same applies to Debian LTS 7 (in the case of debian is a gap already that he may have kernels: hurd, BSD ...)

can remove, my goal was to facilitate the way but was not well seen.